### PR TITLE
Slicing Lab Exercise

### PR DESCRIPTION
**Pull request by Parzival-X**
Completed Slicing Lab exercise.

### CODE REVIEW

**Comment by Parzival-X** [comment]
Completed List Lab exercise.

File: students/rob_sanchez/Lesson 3/list_lab.py [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    # Print list of fruit the user likes
+    list_lenght = len(new_list)
+    if list_lenght == 0:
+        print("Fruit list is empty.")
+    else:
+        print(format_list_output("Fruits you like: ", new_list))
+
+    print()
+
+def series_4():
```
Comment (robalford) [inline students/rob_sanchez/Lesson 3/list_lab.py]: Use two blank lines between module level functions for Pep8

File: students/rob_sanchez/Lesson 3/list_lab.py [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    print(format_list_output("Copy List: ", copy_list))
+    print()
+
+# ***Helper Functions***
+
+# Returns a filtered list of items that begin with the letter "P"
+def get_items_beginning_with_p(list_of_values):
+    filtered_list = []
+    for item in list_of_values:
+        if item.startswith('P') or item.startswith('p'):
```
Comment (robalford) [inline students/rob_sanchez/Lesson 3/list_lab.py]: You can use `if item.lower() == 'p'`

File: students/rob_sanchez/Lesson 3/list_lab.py [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+            filtered_list.append(item)
+    return filtered_list
+
+
+# Removes all instances from a list
+def remove_all_instances(list_of_values, instance):
+    filtered_list = []
+    for item in list_of_values:
+        if instance not in item:
```
Comment (robalford) [inline students/rob_sanchez/Lesson 3/list_lab.py]: I believe this should check equality not membership

File: students/rob_sanchez/Lesson 3/list_lab.py [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+            filtered_list.append(item)
+    return filtered_list
+
+
+# Returns a list with no brackets
+def format_list_output(text, list):
+    return(text + ", ".join(["{}"] * len(list)).format(*list))
```
Comment (robalford) [inline students/rob_sanchez/Lesson 3/list_lab.py]: You don't need the outer parens here

File: students/rob_sanchez/Lesson 3/list_lab.py [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+
+
+# Reverses the letters of each list item
+def reverse_letters(list):
+    reversed = []
```
Comment (robalford) [inline students/rob_sanchez/Lesson 3/list_lab.py]: It's best to avoid using Python built-in variable names like `reversed`

**Pull request merged**